### DISCUSSION TITLE
Moneyball

### DISCUSSION THREAD

**Discussion by mcbgun** [olimorris/PyScoutFM#7] [Ideas]
Hi,

Just thinking that incorporating a moneyball score as well would be useful. FM Stag bases theirs on chalkboard statistics. I'm thinking utilising transfer value and wage to try and find value as well as using your score into the mix to provide an overall moneyball calculation?

**Comment by olimorris** [comment]
I've had a few ideas on how we can really level the app up and Moneyball is one of those ideas. Would love to get your ideas on what statistics we incorporate into this. We could just use Stag's suggestions and use is benchmarking data for what _good_, _average_ and _poor_ look like.

I'd also like to incorporate the ability to rank teams in a league. I'm about to start a Level 8 LLM save in England...I want to know what the best team in the league looks like across GK, DR/DL, DC, MC etc...so I know the quality of players I need to sign.

**Reply by mcbgun** [reply]
I do have an idea....but it involves using two trains of thought. Attributes and statistics. My reasoning is utilising the best of both worlds.....attribute and statistics scoring. Stag had done the groundwork using his spreadsheet which contains the relevant stats and custom calculations. Why not keep your scores as one indicator and incorporate stags metrics as additional indicators so you get...for perhaps the first time I've seen....two confirmations on a player....attributes and stats. I have no idea if there would be a correlation or not..... but would be fascinating. Incidentally I use power bi for work a heck of a lot....I would say I could use that to very quickly determine both median and average attribute/stats for each level with just an additional league filter column to your view. Stats are easy because I believe you don't have to scout them for them to be fully available? Why don't I export all statistics (metrics) from a new save game (I only have vanerama South/North as bottom leagues but we could see what the differences between leagues are and come up with a sensible predicted set of medians/averages? Unless you have a season's worth of data in excel format you can share? To be honest players names become irrelevant as we're checking for baselines so having multiple seasons worth of data will only increase the accuracy.

**Reply by mcbgun** [reply]
In the meantime why not add two simple  calculations....transfer cost per scoring percentage and wage cost per scoring percentage

**Reply by olimorris** [reply]
I have a specific view that moneyball in football does not translate like it does in baseball...but we can still determine if players are undervalued, statistically speaking.

Send over your data export, would love to take a look at it. I'm a little time poor for the next couple of weeks but expect I'll be able to take a look before Jan is over.

**Reply by mcbgun** [reply]
Yes me too....I'll crunch some numbers and send you over a PDF report based on power bi findings. However, I'd need your baseline attributes for each position to filter to just those attributes and your weightings as well so I can apply your logic.

**Reply by olimorris** [reply]
Weightings are all in this repo as a JSON file. They were taken from FM Arena's findings. If you've not read the thread on FM Arena, thoroughly recommend it!
